openapi/components/schemas/Fees/BaseFee.yaml

```
@@ -25,6 +25,24 @@ properties:
       - $ref: ../FilterString.yaml
   formula:
     $ref: ./FeeFormula.yaml
+  settlementSettings:
+    description: |-
+      Fee settlement settings.
+      This value overrides gateway account financial settings of the transaction.
```

suggestion: this sentence is correct without adding `the` here, but I feel that adding it makes the description easier to read
```suggestion
      This value overrides the gateway account financial settings of the transaction.
```